src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -194,6 +194,64 @@ ngraph::pass::GeluFusionWithErfThree::GeluFusionWithErfThree() {
     register_matcher(m, callback);
 }
 
+ngraph::pass::GeluFusionWithErfFour::GeluFusionWithErfFour() {
+    MATCHER_SCOPE(GeluFusionWithErfFour);
+
+    auto input = ngraph::pattern::any_input();
+    auto mul1_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    auto mul1 = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({input, mul1_constant});
+    auto erf = ngraph::pattern::wrap_type<ngraph::opset7::Erf>({mul1});
+    auto mul2_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    auto mul2 = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({erf, mul2_constant});
+    auto add_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    auto add = ngraph::pattern::wrap_type<ngraph::opset7::Add>({add_constant, mul2});
+
+    // x * (0.5 + 0.5 * erf(x * (1 / sqrt(2)))
+    auto mul3 = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({input, add});
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto& pattern_to_output = m.get_pattern_value_map();
+        auto x_output = pattern_to_output.at(input);
+
+        auto mul1_const_value = std::dynamic_pointer_cast<ngraph::opset7::Constant>(
+            pattern_to_output.at(mul1_constant).get_node_shared_ptr());
+        auto add_const_value = std::dynamic_pointer_cast<ngraph::opset7::Constant>(
+            pattern_to_output.at(add_constant).get_node_shared_ptr());
+        auto mul2_const_value = std::dynamic_pointer_cast<ngraph::opset7::Constant>(
+            pattern_to_output.at(mul2_constant).get_node_shared_ptr());
+
+        if (!mul1_const_value || !add_const_value || !mul2_const_value) {
+            return false;
+        }
+
+        bool valid_constant_values = op::util::has_constant_value<float>(mul1_const_value, 1.0f / M_SQRT2, 0.001f) &&
+                                     op::util::has_constant_value<float>(add_const_value, 0.5f) &&
+                                     op::util::has_constant_value<float>(mul2_const_value, 0.5f);
+
+        if (!valid_constant_values) {
+            return false;
+        }
+
+        auto gelu = std::make_shared<ngraph::opset7::Gelu>(x_output);
+
+        gelu->set_friendly_name(m.get_match_root()->get_friendly_name());
+        ngraph::copy_runtime_info(
```

Comment:
please use NodeRegistry class to copy runtime info, #include "node_registry.hpp" (dev_api)
example here https://github.com/openvinotoolkit/openvino/pull/12843/files